app/Filament/Resources/GameResource/RelationManagers/AchievementsRelationManager.php

```diff
@@ -107,7 +98,13 @@ public function table(Table $table): Table
                     ->toggleable(),
             ])
             ->filters([
-
+                Filters\SelectFilter::make('Flags')
+                    ->options([
+                        AchievementFlag::OfficialCore => 'Published',
+                        AchievementFlag::Unofficial => 'Unpublished',
+                    ])
+                    ->default(AchievementFlag::OfficialCore)
+                    ->selectablePlaceholder(false),
```

Comment:
If you remove the Published filter, there's no way to reselect it as it's the default selection in the dropdown:
![image](https://github.com/user-attachments/assets/9bd42488-9202-4733-ad63-03840f6bf687)

You can click Reset to reselect it, but that would also reset any other future filters we add.

You can also select Unpublished, then switch back to Published, but that feel awkward.

Reply:
Should hopefully be a little better in latest.

I've added an "All" option that removes the `Flags` condition from the query that Filament builds up. When you manually removed the "Published" filter, the table falls back to this "All" option, thus making "Published" a selectable value.

![Screenshot 2024-08-17 at 12 54 21 PM](https://github.com/user-attachments/assets/6c8302c6-7122-4248-95e5-8246d96b85fe)